packages/tasks/src/hardware.ts

```diff
@@ -148,6 +148,10 @@ export const SKUS = {
 				tflops: 23.22,
 				memory: [8],
 			},
+			"RTX 5070 Ti": {
+				tflops: 43.9,
+				memory: [16],
+			},
```

Comment:
Spec from https://www.techpowerup.com/gpu-specs/geforce-rtx-5070-ti.c4243


Theoretical Performance

Pixel Rate
    235.4 GPixel/s 

Texture Rate
    686.6 GTexel/s 

FP16 (half)
    43.94 TFLOPS (1:1) 

FP32 (float)
    43.94 TFLOPS 

FP64 (double)
    686.6 GFLOPS (1:64) 



Memory

Memory Size
    16 GB 

Memory Type
    GDDR7

Memory Bus
    256 bit 

Bandwidth
    896.0 GB/s

Reply:
Yeah, we use fp16 as the base reference for GPUs